businesscentral-monitoring/image.yaml

```
@@ -1,20 +1,20 @@
 schema_version: 1
 
 name: "rhpam-7/rhpam-businesscentral-monitoring-rhel8"
-description: "Red Hat Business Central Monitoring 7.13 OpenShift container image"
+description: "IBM Business Automation Manager Open Editions Business Central Monitoring 7.13 OpenShift container image"
 version: "7.13.0"
 from: "registry.redhat.io/ubi8/ubi-minimal:latest"
 labels:
   - name: "com.redhat.component"
-    value: "rhpam-7-businesscentral-monitoring-rhel8-container"
+    value: "ibm-bamoe-7-businesscentral-monitoring-rhel8-container"
   - name: "io.k8s.description"
-    value: "Platform for running Red Hat Business Central Monitoring"
+    value: "Platform for running IBM Business Automation Manager Open Editions Business Central Monitoring"
   - name: "io.k8s.display-name"
-    value: "Red Hat Business Central Monitoring 7.13"
+    value: "IBM Business Automation Manager Open Editions Business Central Monitoring 7.13"
```

we could use only replacement of Red Hat with IBM i think.
Where is the full name use BAMOE but otherwise it is good to replace only necessary refs/names.

```suggestion
    value: "IBM Business Central Monitoring 7.13"
```